dnf/cli/commands/__init__.py

```diff
@@ -711,7 +711,7 @@ def run_on_repo(self):
     class UpgradeSubCommand(SubCommand):
         """Implementation of the upgrade sub-command."""
 
-        aliases = ('upgrade', 'upgrade-to')
+        aliases = ('upgrade', 'upgrade-to', 'up', 'up-to',)
```

Comment:
Upgrade-to is here only due to compatibility (the original command is dead long time ago)

Reply:
Remove `up-to`, please.